### PR TITLE
Update Mortgage datasets and notebook tests dataset path

### PR DESCRIPTION
**Pull request by NvTimLiu**
Suppose there are mortgage/taxi/agaricus sample datasets under the /data(DATA_ROOT) path under customers test cases.

Append mortgage/taxi/agaricus to "DATA_ROOT" dir, then we would not mess up them together,

Otherwise "input" subdir under "DATA_ROOT" would contains all the mortgage/taxi/agaricus datasets,

Or we need to set "DATA_ROOT" diff values one by one pointing to diff subdirs for each of mortgage/taxi/agaricus

Signed-off-by: Tim Liu <timl@nvidia.com>

### CODE REVIEW

**Comment by NvTimLiu** [comment]
@SurajAralihalli Could you please help to take a look, thanks!

**Comment by NvTimLiu** [comment]
`datasets/mortgage-small.tar.gz` is the new sample dataset downloaded following https://github.com/NVIDIA/spark-rapids-examples/blob/branch-22.08/docs/get-started/xgboost-examples/dataset/mortgage.md

Both local verify and CI job got PASS on motgage tests using the new sample dataset

File: examples/XGBoost-Examples/mortgage/notebooks/scala/mortgage-ETL.ipynb [hunk 1]
```diff
@@ -1075,9 +1075,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "XGBoost4j-Spark_new2 - Scala",
+   "display_name": "XGBoost4j-Spark Scala",
    "language": "scala",
-   "name": "xgboost4j-spark_new2_scala"
+   "name": "XGBoost4j-Spark_scala"
```
Comment (NvTimLiu) [inline examples/XGBoost-Examples/mortgage/notebooks/scala/mortgage-ETL.ipynb:1080]: Keep the Jupiter kernel name as what it was

**Comment by SurajAralihalli** [comment]
> `datasets/mortgage-small.tar.gz` is the new sample dataset downloaded following https://github.com/NVIDIA/spark-rapids-examples/blob/branch-22.08/docs/get-started/xgboost-examples/dataset/mortgage.md
> 
> Both local verify and CI job got PASS on motgage tests using the new sample dataset

Hi Tim, I don't think we are allowed to host the Fannie Mae dataset in our repo.

**Comment by viadea** [comment]
@NvTimLiu  Do you want to wait till PR-202 to be merged firstly and then do rebase?

**Comment by NvTimLiu** [comment]
> 

Let me remove the dataset first.

**Comment by NvTimLiu** [comment]
> 

I've rebased it, removed Fannie Mae dataset

**Review by nvliyuan — APPROVED** [review]

**Comment by nvliyuan** [comment]
> BTW, as we are not allowed to host the Fannie Mae dataset in our public repo, do we allow to host it in a private storage?
> 
> Otherwise we need manually download it login to their webpage with username/pw, this is not doable in our CI job.

as discussed, we can host in private, and I will find a way to download FannieMae dataset

**Review by SurajAralihalli — APPROVED** [review]

File: examples/XGBoost-Examples/mortgage/notebooks/scala/mortgage-gpu.ipynb [hunk 2]
```diff
@@ -52,10 +52,10 @@
...
    "source": [
     "// You need to update them to your real paths! The input data files is the output of mortgage-etl jobs\n",
-    "val dataRoot = sys.env.getOrElse(\"DATA_ROOT\", \"/mortgage\")\n",
-    "val trainPath = dataRoot + \"/train/\"\n",
-    "val evalPath  = dataRoot + \"/eval/\"\n",
-    "val transPath = dataRoot + \"/eval/\""
+    "val dataRoot = sys.env.getOrElse(\"DATA_ROOT\", \"/data\")\n",
+    "val trainPath = dataRoot + \"/mortgage/train/\"\n",
+    "val evalPath  = dataRoot + \"/mortgage/eval/\"\n",
+    "val transPath = dataRoot + \"/mortgage/eval/\""
```
[the remark this answers is not in the document]
Comment (NvTimLiu) [reply examples/XGBoost-Examples/mortgage/notebooks/scala/mortgage-gpu.ipynb:58]: Ok, I agree.

Let me update it.

**Review by viadea — APPROVED** [review]

**Comment by NvTimLiu** [comment]
Let me file another PR to add `output/` prefix for the mortgage dataset

**Pull request merged**